ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+
+**DON'T DO THIS!**
+
+This guide is designed to get you thinking about your readers in such a way that you'll end up knowing what docs you need to write.
+It won't take very long for you to get through it, and it will save you time in the long run.
+
+![A smiling person wearing a blue outfit is in a very big hole, with the words "I am stuck" in red](images/stuck.svg "Plz help")
+
+When you're done, you should have a good idea of who your main readers are, the kinds of things they are using your docs for, and what kinds of docs they might be expecting.
+This will help you know which templates to use from the Good Docs Project, and give you the basics to start completing them.
+
+If you want to do this using the interactive tool, go here instead: [Interactive IA Guide](don't click here the link does not exist yet).
+The interactive version is the same as this static version, so you can use this version to check back on later on if you need to.
+
+## What are you writing about?
+
+The first thing to do is think about what you're writing about.
+
+Are you writing about something pretty simple, that people can use without having any experience? Like a toaster?
+
+Or are you writing about something a bit complicated, that people can start to use only after they've done a bit of reading, and might need to ask some questions as they go along? More like a car?
+
+Or are you writing about something really tricky? Something that requires training, a whole lot of background knowledge, and possibly a friend to help them the whole way? Like a nuclear reactor?
+
+**Curse of Knowledge alert!!**
+> Try to remember that not everyone knows the same stuff, or learns in the same way.
+> What if your reader doesn't speak the same language, or is a new migrant from a place with a totally different culture?
+> What if they are nine years old?
+> Or 99 years old?
+> What if they have a physical or mental disability?
+> When you are thinking about your readers, make sure you are thinking about everyone, not just people like you.
+
+### Writing about a toaster
+
+Because the thing you are writing about is pretty simple and straightforward, you might think that you don't need much documentation.
+
+Just because you might not need a lot of docs, you will still need something, and working that out could be tricky.
+You're going to need to think a little bit harder about who you're writing for.
+
+![A silver toaster marked "Super Tooster" with a red discount star, and a sign saying "wowzars!!!"](images/tooster.svg "Super Tooster")
+
+### Writing about a car
+
+There is a pretty good chance that you needed to learn how to do this thing yourself, and hopefully it wasn't all that long ago, and you can fairly easily cast your mind back to those days when you didn't know what this thing was, or how it worked.
+And hopefully there is also someone nearby who has had to learn as well, so go ask them questions!
+
```

Comment:
More concrete types of docs

- A large-leather bound reference volume with complete user-instructions on every aspect of the vehicle functions and specifications (Interior & Exterior).
- ... are there other docs that come with a car?  There might be additional manufacturing documents provided to Car Retailers and Auto-Mechanics that aren't available to customers.